README.md

```diff
@@ -19,4 +33,19 @@ $ npm run start
 $ npm run generate
 ```
 
-For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
```

Comment:
Could you remove the lines starting from "# build for production" until "$ npm run generate"? Those are not really useful for someone new to the project and the build is documented in the [GitHub workflow](https://github.com/TechLabs-Berlin/radar/blob/master/.github/workflows/deploy.yml#L31) anyway.

Reply:
Okay sir